posts/progressive-enhancement/index.qmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Improving Ecosystem Interoperability Iteratively via Progressive Enhancement"
+author:
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-1476"
+date: "2024-07-04"
+categories: [R, interoperability, S3, progressive enhancement, ecosystem, lifecycle, object orientation, DOI]
+format:
+  html: 
+    toc: true
+---
+
+We are continuing our post series on S3 and interoperability.
```

Comment:
In order not to confuse it with other S3's in the programming landscape for example, the [S3 programming language](https://en.wikipedia.org/wiki/S3_(programming_language)#:~:text=S3%20is%20a%20structured%2C%20imperative,for%20its%202900%20Series%20mainframes)
```suggestion
We are continuing our post series on S3 object orientation and interoperability in R.
```